cmd/api/api.go

```diff
@@ -98,9 +95,6 @@ func (c *CmdOpts) startAPI() error {
 			c.controllerHandler(c.caHandler()),
 		)
 	}
-	router.Path(prefix + "/calico/kubeconfig").Methods("GET").Handler(
```

Comment:
Interesting. Why is this Windows specific?

Reply:
Calico needed to be deployed as fully host side thing in Windows, could not be deployed as DaemonSet so we needed some way to get the API access config for it through this endpoint.